.github/workflows/tests.yaml

```diff
@@ -0,0 +1,47 @@
+name: tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  cache-and-install-deps-and-run-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

Comment:
There's a way to reference the version specified in the .nvmrc--see mercury-web's deploy or validate workflows for an example of the exact syntax

Reply:
https://github.com/greatexpectationslabs/mercury-web/blob/main/.github/actions/setup-node/action.yml